### PR TITLE
Add ability to search on solutions statuses

### PR DESCRIPTION
**Pull request by cedric-anne**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

### CODE REVIEW

File: inc/itilsolution.class.php [hunk 1]
```diff
@@ -605,4 +605,44 @@ public function showSummary(CommonITILObject $item) {
...
+
+      if (!is_array($values)) {
+         $values = [$field => $values];
+      }
+
+      switch ($field) {
+         case 'status':
+            $options['display'] = false;
+            $options['value'] = $values[$field];
+            return CommonITILValidation::dropdownStatus($name, $options);
```
Comment (trasher) [inline inc/itilsolution.class.php]: You cannot rely on that here; labels willnot be correct:
- "Waiting for approval" is not correct,
- "Granted" is wrong.

Comment (cedric-anne) [reply inc/itilsolution.class.php]: I agree for "Granted" (I did not saw it as is translated ac "Acceptée" in french), but why "Waiting for approval" is not correct ?

Comment (trasher) [reply inc/itilsolution.class.php]: The "approval" term does not seems to me the best choice here. I do not have anything else to porpose, so if "Granted" has been fixed it's ok for me.

Comment (cedric-anne) [reply inc/itilsolution.class.php]: It has been fixed.

**Pull request merged**